src/lib/components/Navbar.svelte

```diff
@@ -89,86 +89,97 @@ The navbar component.
+            <Dropdown>
+                <svelte:fragment slot="label">
+                    {nextRef.chapter}
+                    <DropdownIcon />
+                </svelte:fragment>
+                <svelte:fragment slot="content">
+                    <TabsMenu
+                        bind:this={chapterSelector}
+                        options={{
+                            Chapter: {
+                                component: SelectGrid,
+                                props: { options: Object.keys(chapters) }
+                            },
+                            Verse: {
+                                component: SelectGrid,
+                                props: { options: Object.keys(chapters[nextRef.chapter]) }
+                            }
+                        }}
+                        active="Chapter"
+                        on:menuaction={navigateReference}
+                    />
+                </svelte:fragment>
+            </Dropdown>
+        {/if}
     </div>
     <div class="dy-navbar-end fill-base-content">
         <!-- Mute/Volume Button -->
-        <button class="dy-btn dy-btn-ghost dy-btn-circle">
-            <label class="dy-swap">
-                <!-- this hidden checkbox controls the state -->
-                <input type="checkbox" bind:checked={$audioActive} />
+        {#if $globalConfig.mainFeatures["audio-turn-on-at-startup"]} 
+            <button class="dy-btn dy-btn-ghost dy-btn-circle">
+                <label class="dy-swap">
+                    <!-- this hidden checkbox controls the state -->
+                    <input type="checkbox" bind:checked={$audioActive} />
 
-                <!-- volume on icon -->
-                <AudioIcon.Volume _class="dy-swap-on fill-black-100" />
+                    <!-- volume on icon -->
+                    <AudioIcon.Volume _class="dy-swap-on fill-black-100" />
 
-                <!-- volume off icon -->
-                <AudioIcon.Mute _class="dy-swap-off fill-black-100" />
-            </label>
-        </button>
+                    <!-- volume off icon -->
+                    <AudioIcon.Mute _class="dy-swap-off fill-black-100" />
+                </label>
+                
+            </button>
+        {/if}
         <!-- Search Button -->
-        <a href="/search" class="dy-btn dy-btn-ghost dy-btn-circle">
-            <SearchIcon />
-        </a>
+        {#if $globalConfig.mainFeatures['search']} 
+            <a href="/search" class="dy-btn dy-btn-ghost dy-btn-circle">
+                <SearchIcon />
+            </a>
+        {/if}
         <!-- Text Appearance Options Menu -->
-        <Dropdown>
-            <svelte:fragment slot="label">
-                <TextAppearanceIcon />
-            </svelte:fragment>
-            <!-- TODO: implement text appearance options -->
-        </Dropdown>
+        {#if $globalConfig.mainFeatures["text-font-size-slider"] && $globalConfig.mainFeatures[ "text-line-height-slider"] }
```

Comment:
This should be an || (or) not an && (and).  Also, need to check whether there is more than one theme enabled.

Reply:
Referring to the theme would it be another comparison to text-font-size-slider and text-line-height-slider?

Reply:
@Jason-Lopez8 You need to get a count of $globalConfig.themes that are enabled.